packages/block-editor/src/store/selectors.js

```diff
@@ -460,6 +460,21 @@ export const getBlockParents = createSelector(
 	( state ) => [ state.blocks.parents ]
 );
 
+/**
+ * Given a block client ID, returns the direct parent.
+ *
+ * @param {Object} state    Editor state.
+ * @param {string} clientId Block from which to find root client ID.
+ *
+ * @return {string} ClientID of the parent block.
+ */
+export const getBlockParent = createSelector(
```

Comment:
**Note:**

After combing through `core/block-editor` selectors, I couldn't find one that would only provide information about the parent, and not the rest of the ancestors. This can be changed if someone has better suggestions.

Reply:
this seems like a legitimate change to me if the behavior doesn't exist yet 👍 

I'm curious how the "select parent block" functionality works without this, though??

Reply:
> I'm curious how the "select parent block" functionality works without this, though??

Great question! I'll look into it. That might give us insight into a selector we can use 🙂

Reply:
The only reference I could find to select parent block was in [entity-record-item.js L42](https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/components/entities-saved-states/entity-record-item.js#L42). It looks like they select the parent block by deriving information about the direct parent from [lines 16 - 29](https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/components/entities-saved-states/entity-record-item.js#L16-L29).

This is an option if we don't want to include a new selector to the block-editor store.